packages/arrayjit/arrayjit.0.3.3/opam

```diff
@@ -30,7 +30,7 @@ depends: [
   "ppx_minidebug" {>= "1.5"}
   "odoc" {with-doc}
 ]
-depopts: ["cudajit"]
+depopts: ["cudajit" {< "0.4.0"}]
```

Comment:
```suggestion
depopts: ["cudajit"]
conflcits: ["cudajit"  {>= "0.4.0"}]
```
Unfortunately depopts do not support bounds, you need to use a conflict for this